src/components/ChangeVotes.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link } from "react-router-dom";
+import axios from "axios";
+
+export default function ChangeVotes() {
+const [alterVotes, setAlterVotes] = useState({})
+
+const {article_id} = useParams()
+//optimistic rendering below
+
+const increaseVotes = (vote) => {
+    patchVotes(vote);
+    console.log(vote, "line 13")
+}
+//optimistic rendering below
+const decreaseVotes = (vote) => {
+    patchVotes(vote);
+    console.log(vote, "line 18")
+}
+
+const patchVotes = (vote) => {
+    axios
+    .patch(`https://be-nc-news-tnfa.onrender.com/api/articles/${article_id}`)
```

Comment:
This is giving an article_id does not exist, it might be worth looking at your back end